source/api/extension/navplace/index.md

```diff
@@ -0,0 +1,331 @@
+
+The `navPlace` property's value follows a specific pattern.
+
+
+
+*   The value _SHOULD_ be an embedded Feature Collection. However, the value _MAY_ be a referenced GeoJSON Feature Collection. 
+*   Feature Collections referenced in the `navPlace` property _MUST_ have the `id` and `type` properties. The reference object _MUST NOT_ have the `features` property, such that clients are able to recognize that it should be retrieved in order to be processed. <br/>
+```json-doc
+{"navPlace":{"id": "https://example.org/iiif/1/feature-collection", "type": "FeatureCollection"}}
+```
+
+
+### 2.2 GeoJSON
+
+This extension utilizes the [GeoJSON specification](https://tools.ietf.org/html/rfc7946). Values for the `navPlace` property follow the conventions for [GeoJSON Feature Collections](https://datatracker.ietf.org/doc/html/rfc7946#section-3.3).
+
+
+#### 2.2.1 GeoJSON as Linked Data
+
+[GeoJSON-LD](https://geojson.org/geojson-ld/) is a publicly available vocabulary and linked data context for the GeoJSON specification. The `navPlace` extension context file contains this context and reference to the vocabulary which ensures the geographic data values are well described. The example below shows how to set the `@context` property for IIIF resources with the `navPlace` property. [Section 3](#3-linked-data) has more detail on linked data compatibility.
+
+
+
+```json-doc
+{
+    "@context":[ 
+       "http://iiif.io/api/extension/navplace/context.json",
+       "http://iiif.io/api/presentation/3/context.json"
+    ]
+}
+```
+
+
+#### 2.2.2 Feature
+
+A Feature represents a spatial bounding. Every Feature is a GeoJSON object. [Go to the GeoJSON specification for a visual example](https://datatracker.ietf.org/doc/html/rfc7946#section-1.5). In this document, the term "Feature" relates to "Feature Object" from the GeoJSON specification.
+
+
+*   A Feature _MUST NOT_ be NULL.
+*   A Feature has a `type` property with the value "Feature".
+*   A Feature has a property with the name `geometry`. The value of the `geometry` property _SHALL_ be either a Geometry object as defined in the [Table of Geometric Shapes](#225-table-of-geometric-shapes) or, in the case that the Feature is unlocated, a JSON null value.
+*   A Feature has a property with the name `properties`. The value of the `properties` property is an object (any JSON object or a JSON null value). For information on using this property to provide information associated with the geographic coordinates, see [Section 3.2](#32-context-considerations-for-geojson-ld-properties).
+*   A Feature _MAY_ have an `id` property. The value of this property _MUST_ be a JSON string or number. If the Feature has a [commonly used HTTP(S) URI identifier](https://iiif.io/api/presentation/3.0/#61-uri-recommendations), it _SHOULD_ be provided as the value of the `id` property. The `id` _MAY_ be the URI of a Feature Collection that contains the Feature with a unique fragment on the end. The Feature _MAY_ be accessible by the URI. 
```

Comment:
**The `id` MUST be a URI. If it's a number (or non URI-containing string) then it's not `id` in the JSON-LD / IIIF sense.**